fggs/fggs.py

```diff
@@ -298,9 +308,11 @@ def remove_edge(self, edge: Edge):
     def copy(self):
```

Comment:
Huh, so a copy doesn't share its labeling with the original graph?

Do you think we might want to add an argument to copy that lets it keep the same labeling dicts?

Reply:
I did have one for a while on one of these classes but took it out because it was not done consistently. It could be put back, but I don't know what it would be used for, so I would lean slightly towards waiting to see it if has a use.